terraform/modules/paas/main.tf

```diff
@@ -59,6 +59,7 @@ resource cloudfoundry_app worker_app {
   instances          = var.worker_app_instances
   memory             = var.worker_app_memory
   space              = data.cloudfoundry_space.space.id
+  strategy           = var.web_app_deployment_strategy
```

Comment:
rename variable to just `deployment_strategy` ?

Reply:
done.